phpBB/phpbb/controller/helper.php

```diff
@@ -22,53 +34,101 @@
 */
 class helper
 {
+	/** @var auth */
+	protected $auth;
+
+	/** @var cache_interface */
+	protected $cache;
+
+	/** @var manager */
+	protected $cron_manager;
+
+	/** @var driver_interface */
+	protected $db;
+
+	/** @var dispatcher */
+	protected $dispatcher;
+
+	/** @var language */
+	protected $language;
+
 	/**
 	* Template object
-	* @var \phpbb\template\template
+	* @var template
```

Comment:
Any change we could update all these docblocks from here downwards aswell to use `/** class */`, lines rather than blocks with double `**`.